public/css/login-registration.css

```diff
@@ -43,73 +56,120 @@ hr {
 }
 
 .login {
-	background-color: transparent;
+	background-color: var(--white);
 	margin-top: 15pt;
 	font-weight: normal;
 	font-size: 14px;
-	color: #50b5dd;
-	border: 1.5px solid #50b5dd;
+	color: #619b83;
+	border: 0px;
 	text-transform: uppercase;
 	border-radius: 10px;
 	margin-bottom: 10pt;
-	padding: 8px 20px;
+	padding: 8px 24px;
 }
 
 .login:hover {
-	background-color: #50b5dd;
+	background-color: var(--green);
 	color: white;
 	cursor: pointer;
 	transition: 0.3s ease;
 }
 
-/* FORM */
+/* TITLE & SUBTITLE */
+.background h1 {
+	margin-top: 0px;
+	margin-bottom: 2px;
+}
 
+.background h2 {
+	margin-top: 4px;
+	margin-bottom: 48px;
+
+	font-size: 30px;
+}
+
+/* FORM */
 .ideahacks-form label {
-	margin-top: 10pt;
-	float: left;
+	display: block;
+	color: var(--black);
+	font-weight: bold;
+
+	margin-top: 12px;
+	margin-bottom: 4px;
+
+	margin-left: calc(10% + 12px);
+
+	text-align: left !important;
```

Comment:
As far as I can tell, you don't need the `!important` here. I guess it won't affect any other pages, so it doesn't really matter, but unnecessary `!important`s make me nervous because they can often mean extra debugging in the future. I leave it up to you if you want to take it out or not; doesn't make a functional difference

Reply:
Yeah, I definitely agree with !important making debugging a lot harder. I've gotten rid of it to make it easier on future developers in dealing with specificity issues. Thanks for catching that!